package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "lint": "node cli.js lint",
-    "test:node": "cross-env AEGIR_TEST=hello node cli.js test -t node",
+    "test:node": "cross-env AEGIR_TEST=hello node cli.js test -t node --files 'test/**/*.spec.js'",
```

Comment:
This should be needed right? As the default will still be `test/**/*.spec.js` as I understand it

Reply:
I think your correct, it looks like that glob runs along with the test/node.js file https://github.com/ipfs/aegir/blob/4b1e37c5f755cc3590d515d39143313ebce7fbf3/src/test/node.js#L32-L35

I'll push an update for this.

Reply:
There is no `test/node.js` so I added this in order to remove the warning it gives. Unnecessary warnings lead to ignoring warnings.

Reply:
Ah yes, @wraithgar that's right, we should leverage the override in package.json to avoid getting that warning. 👍  This should be good to go.  I am excited for grep!